src/components/partner/partner.resolver.ts

```diff
@@ -86,6 +87,11 @@ export class PartnerResolver {
     );
   }
 
+  @ResolveField()
+  growthPartnersClient(@Parent() partner: Partner): SecuredBoolean {
+    return partner.growthPartnersClient ?? partner.globalInnovationsClient;
```

Comment:
I see what you're trying to accomplish here.  I think this works, however, I've never seen the same field both here as a `ResolveField` _and_ on the dto itself as well.  I think you should probably remove it from the dto and keep it here instead during the transition.  Could you make that change and test it that way?

Reply:
Hmmm....I guess I was wrong here.  Ignore this.  🙃

Reply:
Yes, you're correct. The `deprecatedReason` should be declared here too.